css/styles.css

```diff
@@ -89,3 +95,93 @@ max-height: 250px;
   font-size: 2em;
   margin-top: 0;
 }
+
+/* QUOTE */
+
+#review {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  min-height: 600px;
+  background-image: url('../img/food_review.jpg');
+  background-size: cover;
+  background-repeat: no-repeat;
```

Comment:
I don't think that you need this. Background-size:cover should prevent the need for no-repeat.